releaselinks.go

```diff
@@ -33,10 +33,12 @@ type ReleaseLinksService struct {
 //
 // GitLab API docs: https://docs.gitlab.com/ee/api/releases/links.html
 type ReleaseLink struct {
-	ID       int    `json:"id"`
-	Name     string `json:"name"`
-	URL      string `json:"url"`
-	External bool   `json:"external"`
+	ID             int    `json:"id"`
+	Name           string `json:"name"`
+	URL            string `json:"url"`
+	External       bool   `json:"external"`
+	DirectAssetURL string `json:"direct_asset_url,omitempty"`
+	LinkType       string `json:"link_type,omitempty"`
```

Comment:
I think this needs a `const`, like this for example: https://github.com/xanzy/go-gitlab/blob/master/types.go#L36